### PR TITLE
Crt release/1.10.x

### PR DESCRIPTION
**Pull request by 99**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,250 @@
+name: build
+
+on:
+  push:
+    # Sequence of patterns matched against refs/heads
+    branches:
+    # Provide the release branch
+      - crt_release/1.10.x
```
Comment (freddygv) [inline .github/workflows/build.yml:8]: Should this be `release/1.10.x` so that it matches our release branch name?

**Review by freddygv — APPROVED** [review]
LGTM, left one non-blocking question

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+          name: metadata.json
+          path: ${{ steps.generate-metadata-file.outputs.filepath }}
+
+  build:
+    needs: get-product-version
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - {go: "1.17", goos: "linux", goarch: "386"}
```
Comment (freddygv) [inline .github/workflows/build.yml:66]: This should use go 1.16

**Review by freddygv — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 2 above)
Comment (99) [reply .github/workflows/build.yml:66]: updated

File: .github/workflows/build.yml (hunk 1 above)
Comment (99) [reply .github/workflows/build.yml]: updated

**Review by freddygv — APPROVED** [review]

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/498127.

**Pull request merged**